apps/docs-website/docs/migration-guide.mdx

```diff
@@ -74,8 +79,10 @@ export default defineFlatConfig([...sheriff(sheriffOptions)]);
 
 :::info
 
-By default, the [automatic setup process](./setup/automatic-setup.mdx) will not add the `files` in the object and every js/ts file will be linted. Use this only if you want to specifically lint just a subsection of the codebase.
+By default, the [automatic setup process](./setup/automatic-setup.mdx) will not add the `files` in the object
+and every JS/TS file will be linted.
+Only use this if you want to specifically lint just a subsection of the codebase.
 
 :::
 
-- use [eslint-interactive](https://github.com/mizdra/eslint-interactive)
+- Use [`eslint-interactive`](https://github.com/mizdra/eslint-interactive).
```

Comment:
👎

Reply:
I assume the backticks are acceptable though?